README.md

```diff
@@ -232,3 +160,78 @@ The translations are handled by ``src/core/i18n.js``.
 This translation helper that calls the ``@@plonejsi18n`` view defined in plone.app.content to generate a JSON of the translations from the mo file.
 The ``@@plonejsi18n`` view is called one time for a given domain and language and the result is cached in localStorage for 24 hours.
 The only way to test the new translations is to restart the instance to update the mo file from the po file, and then to purge the localStorage to trigger a new download of the translations.
+
+
+### Styleguide
+
+-   Tab: 4 spaces for everything except for HTML and XML files (2 Spaces) and the Makefile (TAB).
+    This rule is defined in the [.editorconfig file](./.editorconfig).
+
+### Commit style guide
+
+We automatically generate the changelog from the commit messages following the [Conventional Commits specification](https://www.conventionalcommits.org/).
+Changelog generation is done by the [conventional changelog plugin](https://github.com/release-it/conventional-changelog/) for [release-it](https://github.com/release-it/release-it).
+This is enforced via a pre-commit hook managed by [husky](https://github.com/typicode/husky).
+
+**And this is how you use it:**
+
+We have 4 different types of changelog entries:
+
+-   Breaking Changes (type: `breaking`),
+-   Features (type: `feat`),
+-   Bug Fixes (type: `fix`),
+-   Maintenance (type: `maint`).
```

Comment:
Other repos also have:

```
-   Documentation (type: `documentation`)
```

`.doc` opens in Word and other office apps by default, and this file extension is explicit.

Is that something you want to add?

Reply:
We could add it in mokcup but we currently reuse `release-it` config from [@patternslib/dev](https://github.com/Patternslib/dev/blob/main/.release-it.js) so it should be extended there first before we mention it here.

@thet what do you think?

Reply:
maybe open an issue for this feature and merge this PR

Reply:
Issue created: https://github.com/plone/mockup/issues/1373